x-pack/legacy/plugins/apm/public/components/app/Settings/SettingsList.tsx

```diff
@@ -294,10 +321,9 @@ export function SettingsList() {
         </EuiCallOut>
 
         <EuiSpacer size="m" />
-
-        {status === 'success' && !hasConfigurations ? (
-          emptyStatePrompt
-        ) : (
+        {status === 'failure' ? (
```

Comment:
Not a super big fan of nested ternaries in terms of readability, might be just me though.

Reply:
I agree. I just didn't have any better suggestions :)